packages/block-library/src/html/block.json

```diff
@@ -5,7 +5,7 @@
 	"attributes": {
 		"content": {
 			"type": "string",
-			"source": "html"
+			"source": "raw"
```

Comment:
I'm pretty sure there's already a PR that does the same thing with lengthy discussion from  @aduth

Reply:
Oh haha :)

Reply:
I didn't find it :(

Reply:
Found it #25120. I was actually just talking with @getdave about this to try to find a solution.

Reply:
I found it, this is the one I was talking about https://github.com/WordPress/gutenberg/pull/10551

Reply:
Hm, this doesn't have any comments from @aduth though.

I'm also not sure that PR is a good solution, because it will break blocks that may use innerHTML differently.

Reply:
@youknowriad Nice, thanks, will read quick.

Reply:
So the difference with that PR is that it still uses the `html` type instead of a new `raw` type, but the solution is essentially the same.

Reply:
yes, and I believe @aduth's point is that it's the `html` type itself that is "entirely" broken

Reply:
Although...if this fixes the immediate experience for users could we land this PR and then write a followup Issue detailing the problems with `html` and how we might look to resolve them long term? Not being able to put HTML in the HTML block feels like a bit of a poor UX 😄

Reply:
I'm not sure we should add a new API (new "source" type)